karaf/images/rhel/Dockerfile

```diff
@@ -30,11 +29,11 @@ RUN echo securerandom.source=file:/dev/urandom >> /usr/lib/jvm/java/jre/lib/secu
 RUN usermod -g root -G jboss jboss
 
 
-# Install Maven via RPM
-ADD "http://download.eng.bos.redhat.com/brewroot/packages/maven3/3.3.3/1.el7/noarch/maven3-${MAVEN_VERSION}.noarch.rpm" /tmp/
-RUN rpm -i "/tmp/maven3-${MAVEN_VERSION}.noarch.rpm" \
- && rm "/tmp/maven3-${MAVEN_VERSION}.noarch.rpm"
+# Install Maven via SCL
 
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms
+
+RUN yum install -y rh-maven33-maven
```

Comment:
Could this be combined into a single `RUN`  with `&& \`  by changing the template ?